src/commands/Admin/eval.js

```diff
@@ -16,17 +16,11 @@ module.exports = class extends Command {
 
 	async run(msg, [code]) {
 		const { success, result, time, type } = await this.eval(msg, code);
-		const footer = this.client.methods.util.codeBlock('ts', type);
+		const footer = this.client.methods.util.codeBlock('ts', type.toString());
```

Comment:
unnecessary, util.codeBlock concatenates the object in a template literal, which automatically calls toString() on the object.

Reply:
Should the typings stay that it expects a string?